azure-pipelines.yml

```diff
@@ -1,14 +1,14 @@
 schedules:
-# Run from Monday to Friday at 5:30 UTC (https://docs.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers?view=azure-devops&tabs=yaml#cron-syntax)
```

Comment:
Why this change? What is actually changed here?

Reply:
Seems like IntelliJ was not intelligent enough and messed up whitespace. I'll fix that. There's IT change that should pop up.

Reply:
Done in the original commit